### DISCUSSION TITLE
How to add sunshine in firewall whitelist (MacOS)

### DISCUSSION THREAD

**Discussion by sky456757** [LizardByte/community#519] [Sunshine]
### Select Topic Area

Question

### Body

I run Sunshine on macOS , but my moonlight client can not connect when firewall is open. Is there any way to add sunshine in mac firewall white list ?

**Comment by cathyjf** [comment]
Which macOS firewall are you talking about? The macOS operating system includes at least two firewall solutions: `pf` and the application firewall. Only the application firewall is enabled by default.

With respect to the application firewall, if `$my_program` is the path to the application you want to add to the whitelist, you can run the following command to add it to the whitelist:

```shell
sudo /usr/libexec/ApplicationFirewall/socketfilterfw --add "$my_program" --unblockapp "$my_program"
```

**Reply by sky456757** [reply]
I try to add /usr/local/bin/sunshine but it not works. I guess there is other program need to add in , but I can not found it

[a comment by an automated account is not shown]
